site/source/tutorials/001_static_content/index.md

```diff
@@ -11,29 +10,25 @@ overview: In this tutorial, you will learn how to the create your first Dojo 2 a
 In this tutorial, you will learn how to the create your first Dojo 2 application and use it to print a simple message in the browser.
 
 ## Prerequisites
-With the Dojo 2 tutorial series, you may work with the examples following two different paths:
+You can [download](./assets/001_static_content-initial.zip) the demo project to get started.
 
-* A browser embedded editor
-* A local installation
+You also need to be familiar with TypeScript as Dojo 2 uses it extensively. For more information, refer to the [TypeScript and Dojo 2](./comingsoon.html) article.
 
-The browser embedded editor is intended to make it easy to quickly see the examples in action, but does not match a normal development environment, and does not provide all of the benefits of TypeScript.
-
-If you prefer a local installation, please visit the [Dojo 2 Installation Guide](../000_local_installation/) before proceeding further with this tutorial.
+## Printing static content
 
-Whether you are working locally or using the embedded editor, you need to be familiar with TypeScript as Dojo 2 uses it extensively. For more information, refer to the [TypeScript and Dojo 2](../comingsoon.html) article.
+Okay, now it is time to start customizing the application. To begin, let's startup the application with the development server so that we can see the impact of our changes. Run the following command in the application's root directory:
```

Comment:
"To begin, let's startup the application with the development server so that we can see the impact of our changes" -> "To begin, let's startup the application with the development server and observe the impact of our changes"